test/dpg/low-level/Expected/AcceptanceTests/DPGCustomizationCustomizedLowLevel/dpgcustomizationcustomizedlowlevel/rest/_request_builders.py

```diff
@@ -169,20 +205,6 @@ def build_get_pages_request(
      `send_request` method. See https://aka.ms/azsdk/python/protocol/quickstart for how to
      incorporate this response into your code flow.
     :rtype: ~azure.core.rest.HttpRequest
-
-    Example:
```

Comment:
didn't get full background yet, just ask that is this Example removed on purpose?

Reply:
yeah, the response is kind of misleading for paging. Also, we don't really ccare about `--low-level-client` (users should use `--version-tolerant` instead really), have been thinking about removing most of our `--low-level-client` tests